src/components/Navigation/NavigationButton.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+import { colorBlack, colorWhite, fontSmall, colorDarkGrey } from '../../styles/designTokens';
+
+const FlexLink = styled(Link)`
+  display: flex;
+  width: 100%;
+  text-decoration: none;
+`;
+
+const LinkContent = styled.span`
+  background: ${({ isActive }) => (isActive ? colorBlack : colorWhite)};
+  border-right: 0px;
```

Comment:
Unnecessary unit

Reply:
As you can see lower, there is a `border` style. I need to reset the size directly for the `border-right` to prevent double border between Links

Reply:
I mean remove only **unit** not whole line ;)
```
  border-right: 0;
```